platform/ui/src/contextProviders/DialogProvider.js

```diff
@@ -169,7 +207,11 @@ const DialogProvider = ({ children, service }) => {
         >
           <div
             id={`draggableItem-${id}`}
-            className={classNames('DraggableItem', isDragging && 'dragging')}
+            className={classNames(
+              'DraggableItem',
+              isDragging && 'dragging',
+              isDraggable && 'draggable'
```

Comment:
classNames supports a nice syntax like:

```js
classNames(
  'DraggableItem',
  {
    'is-dragging': isDragging,
    'is-draggable': isDraggable
   });
```

Your syntax makes sense; just caught me off-guard

Reply:
...me not reading their documentation :eyes: very cool!